index.dd

```diff
@@ -83,11 +83,9 @@ $(DIVID news,
 $(DIVC boxes,
     $(DIVC row,
         $(TOUR newspaper-o, News,
-            $(P $(LINK2 http://dconf.org/2016, DConf 2016) is coming up: May 4-6
-                in Berlin, Germany.
-                $(LINK2 http://dconf.org/2016/registration.html,
-                    Secure your seat)
-                before it's sold out!
+            $(P The D foundation got accepted for the Google Summer of Code 2016
+                and $(LINK https://summerofcode.withgoogle.com/organizations/5078256051027968/, four projects)
```

Comment:
@schuetzm I am really sorry about that. I did this on the run :/
Those d macros are still tricky to me :/